src/git-sync

```diff
@@ -30,15 +30,20 @@ function preconditions {
 
 
 function skip_message_prefix {
-  echo "To skip the sync of the '$(get_current_branch_name)' branch"
+  local branch_name=$(get_current_branch_name)
+  if [ "$(rebase_in_progress)" = true ]; then
```

Comment:
checking whether a rebase is in progress seems like it should belong into `skippable`, not `skip_message_prefix`